src/pages/Admin/ToolsPage/ToolsPage.tsx

```diff
@@ -1,22 +1,20 @@
 import React, { useCallback, useState } from "react";
```

Comment:
Empty undeleted file `src/pages/Admin/ToolsPage/ToolsPage.scss`

Reply:
but it is already deleted. Looks like you haven't noticed. 🤔